attestation-service/attestation-service/src/bin/restful-as.rs

```diff
@@ -49,17 +50,30 @@ enum WebApi {
     Policy,
 }
 
+#[derive(Error, Debug)]
+pub enum RestfulError {
+    #[error("Creating service failed")]
+    Service(#[from] ServiceError),
+    #[error("Failed to read AS config file")]
+    Config(#[from] ConfigError),
+    #[error("Openssl errorstack")]
+    Openssl(#[from] openssl::error::ErrorStack),
+    #[error("io error")]
+    IO(#[from] std::io::Error),
+    #[error("Error")]
+    Anyhow(#[from] anyhow::Error),
```

Comment:
If it is set `#[error("Error")]`, user will see a very simple error string `Error` which will be confusing.

For anyhow, we can use `#[error(transparent)]` due to https://docs.rs/thiserror/1.0.57/thiserror/